gotocompany/optimus/integration/v1beta1/event.proto

```diff
@@ -0,0 +1,60 @@
+syntax = "proto3";
+
+package gotocompany.optimus.core.v1beta1;
+
+import "google/protobuf/timestamp.proto";
+import "gotocompany/optimus/core/v1beta1/resource.proto";
+import "gotocompany/optimus/core/v1beta1/job_spec.proto";
+
+option go_package = "github.com/goto/proton/optimus";
+option java_multiple_files = true;
+option java_outer_classname = "Event";
+option java_package = "com.gotocompany.proton.optimus";
+
+message ResourceChangePayload {
+  string datastore_name = 1;
+  ResourceSpecification resource = 2;
+}
+
+message JobChangePayload {
+  string job_name = 1;
+  JobSpecification jobSpec = 2;
+}
+
+message JobRunPayload {
+  string job_name = 1;
+  string scheduled_at = 2;
```

Comment:
Should the data type for this field be timestamp?

Reply:
Yes, will update the data type.